src/sass/rx/partials/_rx-prescriptions.scss

```diff
@@ -86,17 +85,17 @@ button.rx-prescription-button,
   }
 }
 
+.rx-prescription-count,
 .rx-prescription-action {
-  bottom: 0;
-  position: absolute;
-  right: 0;
+  flex: 1 1 50%;
 }
 
 .rx-prescription-countaction {
-  bottom: -2rem;
-  line-height: 2rem;
-  padding-top: 2rem;
-  position: relative;
+  bottom: 2rem;
+  display: flex;
+  align-items: flex-end;
+  position: absolute;
+  width: 91%;
```

Comment:
This isn't quite accurate when the window size varies; the "padding" gets very slightly off on mobile. If we're using flex here, why not use it for the vertical alignment of as well?